### DISCUSSION TITLE
HTML Template

### DISCUSSION THREAD

**Discussion by RoyiAvital** [jotaen/xit#7] [Tooling]
Could you create (Maybe in Sublime?) an HTML template to recreate the view on your site?
Clicking won't have any meaning but the viewing experience is great.

**Comment by jotaen** [comment]
Glad to head you like it!

I’m not sure what you mean, so what should the view be recreated for? Do you mean as a Sublime Text colour scheme?

**Reply by RoyiAvital** [reply]
I would like to be able to generate a single HTML page like [your site](https://xit.jotaen.net/) form my `xit` file.

**Reply by jotaen** [reply]
Ah I see. Currently, there is no parsing tool for the file format yet. But maybe someone writes one, that e.g. is capable of outputting JSON or some other general-purpose format, which then can be used for that purpose.

By the way: the viewing experience with a good editor plugin should come pretty close. I hope [the one for Sublime Text](https://github.com/jotaen/xit-sublime) will be available soon via the built-in package manager (hopefully in a few weeks). Maybe there will be other people who are willing to contribute plugins for other editors too.

**Reply by RoyiAvital** [reply]
The editor will be great. But sharing with others (The manager?) in a single HTML file would be better.
I think it would be great if you could make your Sublime Text plug in to be able to generate those HTML's.

**Reply by kilicbaran** [reply]
The author of the discussion wants a static HTML page, but I want to add that a web-based UI can be useful for doing things like sorting by importance or due date without actually changing the order in the file, or easily ordering multiline tasks.

**Reply by jotaen** [reply]
Yes, such tooling support will definitely be helpful. The idea is to keep the file specification separate from the tools, so that anyone could build these kinds of interfaces on top of the standardised `.xit` file format.
